server/package.json

```diff
@@ -1,26 +1,14 @@
 {
   "name": "@firebolt-js/mock-firebolt",
-  "version": "2.0.1",
+  "version": "1.0.1",
   "description": "Controllable mock Firebolt server",
   "main": "./build/index.mjs",
   "scripts": {
     "clean": "node ./scripts/clean.mjs",
-    "prepare:setup": "npm run clean",
-    "build:discovery": "node ./scripts/copy.mjs ./node_modules/@firebolt-js/discovery-sdk/dist/firebolt-discovery-open-rpc.json ./build/firebolt-discovery-open-rpc.json",
-    "build:openrpc:NOTES": "Only use if you are using all three Firebolt SDKs",
-    "build:openrpc": "npm run build:discovery",
-    "build:mf": "node ./scripts/rcopy.mjs ./src ./build && node ./scripts/copy.mjs ./src/.mf.config.json ./build/.mf.config.json && node ./scripts/copy.mjs ./package.json ./build/package.json",
-    "build:NOTES": "Only use if you are using all three Firebolt SDKs",
-    "build": "npm run build:openrpc && npm run build:mf",
+    "build": "node ./scripts/rcopy.mjs ./src ./build && node ./scripts/copy.mjs ./src/.mf.config.json ./build/.mf.config.json && node ./scripts/copy.mjs ./package.json ./build/package.json",
     "test": "NODE_OPTIONS=--experimental-vm-modules NODE_ENV=test npx jest --config=jest.config.js --detectOpenHandles --runInBand --forceExit",
-    "start:NOTES1": "Can do 'npm run start' (core/manage OpenRPC only) or 'npm run start -- --discovery'",
-    "start": "node build/index.mjs",
-    "dev:NOTES1": "Typical developer script once 'npm run build:openrpc' has been executed once",
-    "dev:NOTES2": "Can do 'npm run dev' (core/manage OpenRPC only) or 'npm run dev -- --discovery'",
-    "dev": "npm run build:mf && npm run start --",
```

Comment:
Can we add this back for backwards compatibility? I guarantee no one will read that things have changed and they'll just "npm run dev" as usual.

Can we update it to:
* Echo a deprecation warning and mention it may be removed in a future release
* Redirect to npm start

Reply:
Add the run dev command back with a deprecation warning included